src/Testcontainers.AspireDashboard/AspireDashboardBuilder.cs

```diff
@@ -0,0 +1,91 @@
+namespace Testcontainers.AspireDashboard;
+
+/// <inheritdoc cref="ContainerBuilder{TBuilderEntity, TContainerEntity, TConfigurationEntity}" />
+[PublicAPI]
+public sealed class AspireDashboardBuilder : ContainerBuilder<AspireDashboardBuilder, AspireDashboardContainer, AspireDashboardConfiguration>
+{
+    public const string AspireDashboardImage = "mcr.microsoft.com/dotnet/aspire-dashboard:latest";
```

Comment:
```suggestion
    public const string AspireDashboardImage = "mcr.microsoft.com/dotnet/aspire-dashboard:8.0";
```

Can we please pin the version?

Reply:
Good catch, thank you